gnovm/pkg/gnolang/frame.go

```diff
@@ -39,7 +39,7 @@ func (fr Frame) String() string {
 			fr.NumExprs,
 			fr.NumStmts,
 			fr.NumBlocks,
-			fr.LastPackage.PkgPath,
+			fr.LastPackage.ModFile.Path,
```

Comment:
We should not implement mod and package versioning at the VM level. The module is a development tool that manages package versioning outside of VM execution.